### ISSUE TITLE
skimage.transform.resize: anti_aliasing_sigma default value

### ISSUE THREAD

**Issue by etiennelndr** [scikit-image/scikit-image#3890]
## Description
I was reading the documentation of the [**skimage.transform.resize**](https://github.com/scikit-image/scikit-image/blob/51d4fd9294955deee48491a5ae4c01c1dfe0f48d/skimage/transform/_warps.py#L19) function when I found that, when `anti_aliasing_sigma` is set to `None`, this parameter should have the default value ["(1 - s) / 2"](https://github.com/scikit-image/scikit-image/blob/51d4fd9294955deee48491a5ae4c01c1dfe0f48d/skimage/transform/_warps.py#L68). But, [line 107](https://github.com/scikit-image/scikit-image/blob/51d4fd9294955deee48491a5ae4c01c1dfe0f48d/skimage/transform/_warps.py#L107) this parameter is set to `np.maximum(0, (factors - 1) / 2)`. Shouldn't it be `np.maximum(0, (1 - factors) / 2)` ?

## Version information
```python
from __future__ import print_function
import sys; print(sys.version)
import platform; print(platform.platform())
import skimage; print("scikit-image version: {}".format(skimage.__version__))
import numpy; print("numpy version: {}".format(numpy.__version__))
```

```python
3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 03:37:03) [MSC v.1900 64 bit (AMD64)]
Windows-7-6.1.7601-SP1
scikit-image version: 0.14.2
numpy version: 1.14.5
```

This error is also in the current version ([see here](https://github.com/scikit-image/scikit-image/blob/51d4fd9294955deee48491a5ae4c01c1dfe0f48d/skimage/transform/_warps.py#L107)).

**Comment by hmaarrfk** [comment]
He thanks for reporting this apparent discrepancy.

I don't think there is a real difference between

```
(factor - 1) / 2
```
and
```
(1 - factor) /2
```
The reason is that `sigma` is typically considered as the `standard deviation` which is the `square root of the variance`, a positive real number.

I think the documentation should be updated to reflect the implementation, which assumes `factor > 1`, making `(factors - 1) / 2` a positive number.

Would you like to submit the pull request making this change to the documentation?

**Comment by etiennelndr** [comment]
First of all I'm sorry for the late reply. To return to the main subject, I totally agree with you and I'll submit a pull request today.

**Comment by hmaarrfk** [comment]
Fixed in #3911
